src/drivers/intel/cavs/sue-ipc.c

```diff
@@ -70,6 +70,8 @@ void ipc_platform_send_msg(struct ipc_msg *msg)
 	platform_shared_commit(ipc, sizeof(*ipc));
 
 	spin_unlock_irq(&ipc->lock, flags);
+
+	return 0;
 }
```

Comment:
On Sue there is a queue of messages with no interrupts or something or this platform is still incomplete?

Reply:
SUE communicates mostly by UART

Reply:
A bit odd in my opinion but okay :))